tests/helper/helper_oc.go

```diff
@@ -274,6 +275,13 @@ func (oc *OcRunner) checkForImageStream(name string, tag string) bool {
 	return false
 }
 
+// ImportSupportedImage import all the supported images
+func (oc *OcRunner) ImportSupportedImage(registry, image, cmpType, project string) {
```

Comment:
@prietyc123 The name looks like it only works for support images but actually it does not. As per the args passed, It imports any image type into the namespace. So instead of making the function name specific you can use name something like ```ImportImageTypeIntoProject``` instead.

Reply:
> @prietyc123 The name looks like it only works for support images but actually it does not. As per the args passed, It imports any image type into the namespace. So instead of making the function name specific you can use name something like `ImportImageTypeIntoProject` instead.

How about `ImportImageFromRegistry` ?

Reply:
> How about ImportImageFromRegistry ?

This makes more sense